module/VuFind/src/VuFind/Search/Minified.php

```diff
@@ -74,20 +74,54 @@ class Minified
     public $hf = [];
 
     /**
-     * ID, start tIme, query Speed, Result total, search TYpe, search CLass id
+     * Search ID, start tIme, query Speed, Result total, search TYpe, search CLass id
```

Comment:
This comment can be trimmed down to only reference the id.

Reply:
Fixed.